### PR TITLE
User can create an entry

### PR DESCRIPTION
**Pull request by t-doan**
user-can-create-an-entry

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,12 +16,50 @@
...
+    <header class="header-color">
+      <div class="container">
+        <h1 class="margin-none header-padding">Code Journal</h1>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
-      <div data-view="entries" class="hidden"></div>
+      <form id="entry-form">
+        <div data-view="entry-form">
```
Comment (shawnkost) [inline index.html]: These should be swapped. The `form` should be inside the `div`

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/* global data */
+
+// console.log(data);
```
Comment (shawnkost) [inline js/main.js]: Remove all old console logs

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  const entryValues = {
+    title: $formSubmit.elements.title.value,
+    image: $formSubmit.elements.photo.value,
+    note: $formSubmit.elements.notes.value,
+    entryId: data.nextEntryId,
+  };
+
+  data.nextEntryId++;
+  data.entries.push(entryValues);
```
Comment (shawnkost) [inline js/main.js]: You want to add this to the beginning of the array rather than the end

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good so far Tai! A few changes here

File: index.html [hunk 4]
```diff
@@ -22,8 +22,8 @@ <h1 class="margin-none header-padding">Code Journal</h1>
       </div>
     </header>
     <main class="container">
-      <form id="entry-form">
-        <div data-view="entry-form">
+      <div data-view="entry-form">
+        <form id="entry-form">
```
Comment (t-doan) [inline index.html:26]: made the changes

File: index.html [hunk 5]
```diff
@@ -57,9 +57,9 @@ <h1>New Entry</h1>
           <div class="row justify-content-end">
             <input type="submit" id="save-button" name="save" value="save" />
           </div>
-        </div>
-        <div data-view="entries" class="hidden"></div>
-      </form>
+        </form>
+      </div>
+      <div data-view="entries" class="hidden"></div>
```
Comment (t-doan) [inline index.html:62]: made the changes

File: js/main.js [hunk 6]
```diff
@@ -1,7 +1,5 @@
 /* global data */
 
-// console.log(data);
-
```
Comment (t-doan) [inline js/main.js]: removed the console.log

File: js/main.js [hunk 7]
```diff
@@ -24,7 +22,7 @@ function handleSubmit(event) {
   };
 
   data.nextEntryId++;
-  data.entries.push(entryValues);
+  data.entries.unshift(entryValues);
```
Comment (t-doan) [inline js/main.js:25]: changed to `unshift` method instead of `push`

Comment (t-doan) [reply js/main.js:25]: New entries will be the first index of the array

image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**